docs/appendix/server-datasets.md

```diff
@@ -47,8 +48,9 @@ ZWENOKRR | JCL | Removes key ring and certificates for RACF
 ZWENOKRT | JCL | Removes key ring and certificates for TSS
 ZWENOKYR | JCL | Removes key ring and certificates
 ZWENOSEC | JCL | Defines security permits
-ZWERMVS | JCL | Removes datasets used by a Zowe instance
-ZWERMVS2 | JCL | Removes the APF load library
+ZWERMVS | JCL | Removes PARMLIB dataset
+ZWERMVS1 | JCL | Removes plugin dataset
```

Comment:
Hi Martin, so as per Zowe Docs standards, we should change all references of dataset to two words (data set) and plugin to plug-in. I will do a find and replace of these references and push the changes but I wanted you to know for future reference. This is just for consistency across Zowe Docs.